ui/kotlinx-coroutines-android/build.gradle

```diff
@@ -12,6 +23,66 @@ dependencies {
 
     testImplementation 'com.google.android:android:4.1.1.4'
     testImplementation 'org.robolectric:robolectric:4.0-alpha-3'
+    testImplementation 'org.smali:baksmali:2.2.7'
+
+    // TODO Replace with a 1.6.x version once released to maven.google.com.
+    r8 'com.android.tools:r8:a7ce65837bec81c62261bf0adac73d9c09d32af2'
+}
+
+class RunR8Task extends JavaExec {
+
+    File outputDex
+    File inputConfig
```

Comment:
It seems that those files shall marked with Input/Ouput gradle annotations, because otherwise this task is not incremental and is being rerun every time even if nothing has changed.